homeassistant/components/light/tradfri.py

```diff
@@ -297,11 +297,15 @@ def _refresh(self, light):
         self._rgb_color = None
         self._features = SUPPORTED_FEATURES
 
-        if self._light_data.hex_color is not None:
-            if self._light.device_info.manufacturer == IKEA:
+        if self._light.device_info.manufacturer == IKEA:
```

Comment:
What do you think of pushing this knowledge into pytradfri ? That way any consumer of that lib can leverage this knowledge. Maybe we can add `support_rgb` and `support_color_temp` properties to the Device class?

Reply:
Example of existing properties that do something similar: https://github.com/ggravlingen/pytradfri/blob/master/pytradfri/device.py#L49-L52

Reply:
I'm working on it - moving this and also max and min temperatures, as they differ for colour and white spectrum bulbs. But since it is first-pytradfri-than-ha change so we wanted allow users to play with their RGB Tradfris earlier.

Reply:
I'm the author of both, so let's just get pytradfri up to date and we can get in for 0.56?

Reply:
OK, since 0.55 just flew by (-: